subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -157,14 +157,26 @@ static void handle_wifi_twt_event(struct net_mgmt_event_callback *cb)
 		print(context.sh, SHELL_NORMAL, "TWT response: %s for dialog: %d and flow: %d\n",
 		      wifi_twt_setup_cmd2str[resp->setup_cmd], resp->dialog_token, resp->flow_id);
 
-		/* If accepted, then no need to print TWT params */
-		if (resp->setup_cmd != WIFI_TWT_SETUP_CMD_ACCEPT) {
-			print(context.sh, SHELL_NORMAL,
-			      "TWT parameters: trigger: %s wake_interval: %d us, interval: %lld us\n",
-			      resp->setup.trigger ? "trigger" : "no_trigger",
-			      resp->setup.twt_wake_interval,
-			      resp->setup.twt_interval);
-		}
+		print(context.sh, SHELL_NORMAL, "negotiation_type: %s\n",
```

Comment:
We already are dumping TWT info https://github.com/zephyrproject-rtos/zephyr/blob/ca330e04e72a8330b8f48badb02048483c98c82d/subsys/net/l2/wifi/wifi_shell.c#L478 so, better to move that to a helper function and use it here.